ddocs/medic-client/views/tasks/map.js

```diff
@@ -0,0 +1,14 @@
+function(doc) {
+  if (doc.type === 'task') {
+    if (doc.owner) {
+      var isTerminalState = ['Cancelled', 'Completed', 'Failed'].indexOf(doc.state) >= 0;
+      if (!isTerminalState) {
+        emit('owner-' + doc.owner, { _id: doc.id });
+      }
+    }
+
+    if (doc.requester) {
+      emit('requester-' + doc.requester, { _id: doc.id });
```

Comment:
You're emitting a value of `{ _id: doc.id }` which is bad for performance, and as far as I can tell unused.